default/python/turn_events/turn_events.py

```diff
@@ -6,6 +6,66 @@
 from universe_tables import MONSTER_FREQUENCY
 
 
+
+# tuples of galaxy setup options to pick from randomly
+low_to_high =  (fo.galaxySetupOption.low, fo.galaxySetupOption.medium, fo.galaxySetupOption.high)
+none_to_high = (fo.galaxySetupOption.low, fo.galaxySetupOption.medium, fo.galaxySetupOption.high, fo.galaxySetupOption.none)
```

Comment:
the order of the items is irrelevant; it just needs to pick one. the name is was chosen to convey that all, or all but none, of the possible values were included.